plutus-core/cost-model/data/builtinCostModelA.json

```diff
@@ -1222,10 +1222,17 @@
   "valueContains": {
     "cpu": {
       "arguments": {
-        "intercept": 1163050,
-        "slope": 1470
+        "constant": 278039,
+        "model": {
+          "arguments": {
+            "intercept": 386227,
+            "slope1": 1970,
```

Comment:
The  theoretical linear upper bound of `k*(m+n)` would have `slope1 = slope2` here, but here the two slopes are very different!  I think that's OK though: the theoretical bound is very conservative and this seems to fit the data very well.